file_test.go

```diff
@@ -40,8 +52,30 @@ func TestDownloadGitTagZipFile(t *testing.T) {
 			GitTag: tc.gitTag,
 		}
 
-		zipFilePath, err := downloadGithubZipFile(gitHubCommit, tc.githubToken)
+		zipFilePath, err := downloadGithubZipFile(gitHubCommit, tc.githubToken, tc.instance)
+
 		defer os.RemoveAll(zipFilePath)
+
+		// We don't have a running instance of GitHub Enterprise against which to validate tests as we do for GitHub public,
+		// so this test will only validate that fetch attempted to download from the expected URL. The download itself
+		// will fail.
+
+		githubEnterpriseDownloadUrl := fmt.Sprintf("https://%s/repos/%s/%s/zipball/%s", tc.instance.ApiUrl, tc.repoOwner, tc.repoName, tc.gitTag)
+
+		// TODO: The awkwardness of this test makes it clear that a better structure for this program would be to refactor
+		// the downloadGithubZipFile() function to a function called downloadGithubFile() that would accept a URL as a
+		// param. We could then test explicitly that the URL is as expected, which would make GitHub Enterprise test cases
+		// simpler to handle.
```

Comment:
You could run a dummy http server as part of the test (see [example in Terratest](https://github.com/gruntwork-io/terratest/blob/master/modules/http-helper/dummy_server.go)) and use localhost as the URL you pass to fetch...

Reply:
That's a good idea, but just too much work for the available time I had to work on this.